src/client/ContentManagement.js

```diff
@@ -612,7 +612,8 @@ exports.CreateContentObject = async function({libraryId, objectId, options={}})
     path: path,
     body: {      // filter out options not recognized by server (noEncryptionConk, createKMSConk)
       type: options.type,
-      meta: options.meta
+      meta: options.meta,
+      copy_from: options.copy_from
     }
```

Comment:
We already have a function CopyContentObject()
- https://eluv-io.github.io/elv-client-js/module-ElvClient_ContentManagement.html#.CopyContentObject

I'm fine adding an option to the create but is it useful to have two ways of doing the same thing?

(We should specify that in the comments (the docs pick up the  function header comments)